tests/dummy/app/controllers/browsers.js

```diff
@@ -47,12 +44,9 @@ export default Ember.Controller.extend({
         }, {
             name: 'modal',
             lib: { 'twb': true }
-        }, {
-            name: 'pagination',
```

Comment:
This is not related to the `sl-panel` component.  Is this modification intentional?